code/datums/skills/cleaning.dm

```diff
@@ -0,0 +1,4 @@
+/datum/skill/cleaning
+	name = "Scrubbing"
+	desc = "It’s not who I am underneath, but what I mop up that defines me."
+	modifiers = list(SKILL_SPEED_MODIFIER = list(1.15, 1.05, 1, 0.9, 0.75, 0.5, 0.36))
```

Comment:
im not sure if i like the use of skill_Speed_modifier if you're using it for things like efficiency of soap uses etc

Reply:
I can leave a comment that it handles nonspeed stuff as well, I don't think there's much to gain by making an equivalent second list until I feel like they need to be different values.